assets/src/js/Components/ArchivePicker.js

```diff
@@ -246,6 +258,20 @@ export default function ArchivePicker() {
       <div className={classNames('image-picker', {'open-sidebar': selectedImagesAmount > 0})}>
         <ArchivePickerList />
 
+        {images.length > 0 && (
+          <div className="help">
+            <div
+              className="tooltip"
+              dangerouslySetInnerHTML={{
+                __html: __(
```

Comment:
I'm not sure if I understand why you use `dangerouslySetInnerHTML` instead of just adding content into the `div`.

Reply:
If I don't use `dangerouslySetInnerHTML` the HTML tags are shown (probably because it's in a translation)